modules/core/arrow-test/src/main/kotlin/arrow/test/laws/ApplicativeLaws.kt

```diff
@@ -12,42 +12,42 @@ import io.kotlintest.properties.forAll
 object ApplicativeLaws {
 
   fun <F> laws(A: Applicative<F>, EQ: Eq<Kind<F, Int>>): List<Law> =
-    FunctorLaws.laws(A, EQ) + listOf(
-      Law("Applicative Laws: ap identity") { A.apIdentity(EQ) },
-      Law("Applicative Laws: homomorphism") { A.homomorphism(EQ) },
-      Law("Applicative Laws: interchange") { A.interchange(EQ) },
-      Law("Applicative Laws: map derived") { A.mapDerived(EQ) },
-      Law("Applicative Laws: cartesian builder map") { A.cartesianBuilderMap(EQ) },
-      Law("Applicative Laws: cartesian builder tupled") { A.cartesianBuilderTupled(EQ) }
-    )
+          FunctorLaws.laws(A, EQ) + listOf(
```

Comment:
format

Reply:
Fixed! I updated this branch to be aligned with master and used ktlintFormat gradle task 🤔 I thought our ktlint config would fix this, however, I had to fix these files manually.